.travis.yml

```diff
@@ -10,6 +10,9 @@ env:
   global:
   - RUBYOPT='-W0'
   - COVERAGE=1
+  matrix:
+  - INTEGRATION_SPECS=0
+  - INTEGRATION_SPECS=1
```

Comment:
For clarification: "Deployment needs fixing." =~ "A condition needs to be added to the `deploy:` section to ensure it only runs for one build matrix entry."